frontend/views/pages/GroupChat.vue

```diff
@@ -190,7 +190,7 @@ export default ({
       })
       if (this.isDirectMessage(chatRoomId)) {
         this.updateCurrentChatRoomID(chatRoomId)
-      } else if (chatRoomId && chatRoomId !== this.currentChatRoomId) {
+      } else if (chatRoomId) {
```

Comment:
I'm currently working on fixing a merge conflict with [this](https://github.com/okTurtles/group-income/pull/1463/files#diff-bdfc5336979391ea372a8f374feff06a47c882e04271fec2f3186bd98a340341R190).

What was the reason this change was made?

I'm leaning toward simply accepting the changes from Alex's DMs PR over this, but if there was a good reason for this would like to know.

Reply:
@Silver-IT maybe you can comment here as to the purpose of this `&& chatRoomId !== this.currentChatRoomId` ?

Reply:
It's nice catch, actually. As I added debounce to `refreshContent` function in `ChatMain.vue`, this is not needed.
I have found more unnecessary codes in this `watch`, so I am going to update this in the next PR which I am currently working on. It's for fixing Issue #1493.

Reply:
I want you to just follow the current master branch so that I can optimize it later.